Projects/ChangeTier/Tests/ViewModelTests/ChangeTierViewModelTests.swift

```diff
@@ -0,0 +1,214 @@
+import XCTest
+import hCore
+
+@testable import ChangeTier
+@testable import hCoreUI
+
+final class ChangeTierViewModelTests: XCTestCase {
+    weak var sut: MockChangeTierService?
+
```

Comment:
can we also add
weak var vm: ChangeTierViewModel? 
and set value to it and make sure its nil when we finish each test